src/test/java/seedu/address/testutil/TypicalIncomes.java

```diff
@@ -18,5 +29,35 @@ public class TypicalIncomes {
             .withDate(LocalDateTime.of(2023, 10, 20, 0, 0))
             .build();
 
+    public static final Income TUITION = new IncomeBuilder().withName("Tuition")
+            .withAmount(450.0)
+            .withDate(LocalDateTime.of(2023, 11, 25, 12, 12))
+            .build();
+
+    // Manually added - Income's details found in {@code CommandTestUtil}
+    public static final Income NUS = new IncomeBuilder().withName(VALID_INCOME_NAME_NUS)
+            .withAmount(VALID_AMOUNT_NUS)
+            .withDate(VALID_DATETIME_NUS)
+            .build();
+    public static final Income INTERN = new IncomeBuilder().withName(VALID_INCOME_NAME_INTERN)
+            .withAmount(VALID_AMOUNT_INTERN)
+            .withDate(VALID_DATETIME_INTERN)
+            .build();
+
     private TypicalIncomes() {} // prevents instantiation
+
+    /**
+     * Returns an {@code IncomeList} with all the typical persons.
+     */
+    public static IncomeList getTypicalIncomeList() {
+        IncomeList ab = new IncomeList();
+        for (Income income : getTypicalIncomes()) {
+            ab.addIncome(income);
+        }
```

Comment:
Minor nit but could we not use the `resetData` function for this? Though I think that might be overkill, anything works for me in this case

Reply:
That's for the abstraction layer below the model, unless we expose another method